primitives/src/ethereum_checked.rs

```diff
@@ -35,6 +35,8 @@ use sp_std::{prelude::*, result::Result};
 /// Max Ethereum tx input size: 65_536 bytes
 pub const MAX_ETHEREUM_TX_INPUT_SIZE: u32 = 2u32.pow(16);
 
+pub type EthereumTxInput = BoundedVec<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the `EthereumTxInput` type and its usage in the `CheckedEthereumTx` struct is a good approach to encapsulate the Ethereum transaction input. However, it's important to validate the usage of this new type in the wider context of the codebase. Ensure that this change doesn't introduce any data validation issues or unexpected behaviors when handling Ethereum transaction inputs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->